sebs.py

```diff
@@ -237,9 +240,15 @@ def invoke(
     if timeout is not None:
         benchmark_obj.benchmark_config.timeout = timeout
 
+    function_name = function_name if function_name else deployment_client.default_function_name(benchmark_obj)
+
+    # GCP: augment function name with trigger type: _http, _queue etc.
+    if deployment_client.name() == "gcp" or deployment_client.name() == "azure":
```

Comment:
We should add somewhere a comment - GCP can only have one trigger per function, and Azure requires defining triggers at deployment time.